virtual_rainforest/models/hydrology/hydrology_model.py

```diff
@@ -801,3 +876,147 @@ def calculate_drainage_map(grid: Grid, elevation: np.ndarray) -> dict[int, list[
     upstream_ids = find_upstream_cells(lowest_neighbours)
 
     return dict(enumerate(upstream_ids))
+
+
+def estimate_interception(
```

Comment:
I wonder given the size of this file (and the fact the model is likely to get more complex) whether it is worth moving some of these functions to a separate submodules like maybe `virtual_rainforest.hydrology.above_ground` or something

Reply:
I was going to say the same thing....